src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/RealTimeLength/RealTimeLength.h

```diff
@@ -0,0 +1,23 @@
+#ifndef REALTIMELENGTH_H_INCLUDED
+#define REALTIMELENGTH_H_INCLUDED
+
+#ifndef SENSOR_DISTANCE
+#define SENSOR_DISTANCE 3.178
+#endif
+
+class RealTimeLength{

```

Comment:
`sstack` and `ssindex` seem somehow related - they should be coupled together. There seems to be no real reason to use fix sized array of pointers - somehow clear to code so it does not look like plain C.